pkg/blobstore/configuration/new_blob_access.go

```diff
@@ -385,7 +387,17 @@ func NewNestedBlobAccess(configuration *pb.BlobAccessConfiguration, creator Blob
 		if err != nil {
 			return nil, err
 		}
-		implementation = blobstore.NewReadFallbackBlobAccess(primary, secondary)
+		var replicator replication.BlobReplicator
```

Comment:
Current style is not to rely on default configuration options too heavily. Just remove the if-condition below and call into NewBlobReplicatorFromConfiguration() unconditionally.

Reply:
Sure - I'll note that this does break existing configurations, I was just trying to make it backwards compatible.